src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/operation/ExecuteTaskReconfigurationService.java

```diff
@@ -144,6 +145,18 @@ private List<String> reconfigureTasks(List<String> taskIds, List<TaskResource> s
         return failedTaskIds;
     }
 
+    private void resetIndexed(TaskResource taskResource) {
+        log.info("indexed attribute for the task (id={}) before change is {} ",
+                 taskResource.getTaskId(), taskResource.getIndexed());
+        if (!taskResource.getIndexed()
```

Comment:
can we dd unit tests to cover the lines in the if condition? I.e. !taskResource.getIndexed() (line 151) & taskResource.getState() == CFTTaskState.UNASSIGNED (line 153).

See here fore reference -> https://sonarcloud.io/component_measures?metric=new_coverage&selected=uk.gov.hmcts.reform%3Awa-task-management-api%3Asrc%2Fmain%2Fjava%2Fuk%2Fgov%2Fhmcts%2Freform%2Fwataskmanagementapi%2Fservices%2Foperation%2FExecuteTaskReconfigurationService.java&view=list&pullRequest=1119&id=uk.gov.hmcts.reform%3Awa-task-management-api

Reply:
Hi Ben, I've added the test cases as discussed. Could you please verify and resolve this conversation.